spec/acceptance/instance_spec.rb

```diff
@@ -1,11 +1,13 @@
 require 'spec_helper_acceptance'
 require 'securerandom'
 require 'ipaddr'
+require 'pry'
```

Comment:
probably best left out, only used for debugging and might not be in the Gemfile for running the acceptance tests